### PR TITLE
UF-10231: Added sibling resources with request parameters

### PR DESCRIPTION
**Pull request by josc04**
- Added sibling resources with request parameters as some parts in Open-E cant handle path parameters

## Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Removed feature
- [ ] Code style update (formatting etc.)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?
- [ ] Yes (I have stepped the version number accordingly)
- [x] No
      
## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly (if applicable).
- [x] I have added/updated tests to cover my changes (if applicable).

### CODE REVIEW

File: src/integration-test/resources/openeIT/__files/test06_getTypeForExistingErrandWithRequestParameter/mappings/api-byggr-read-errand.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          "equalToXml" : "<?xml version=\"1.0\" encoding=\"UTF-8\" ?><SOAP-ENV:Envelope xmlns:SOAP-ENV=\"http://schemas.xmlsoap.org/soap/envelope/\"><SOAP-ENV:Header/><SOAP-ENV:Body><GetArende xmlns=\"www.tekis.se/ServiceContract\"><dnr>BYGG 2024-000666</dnr></GetArende></SOAP-ENV:Body></SOAP-ENV:Envelope>"
+	    }]
+	},
+	"response": {
+		"headers": {
+			"Content-Type": "application/xml"
+		},
+		"status": 200,
+		"bodyFileName": "test06_getTypeForExistingErrandWithRequestParameter/responses/api-byggr-read-errand-response.xml"
+		}
```
Comment (foobar-1337) [inline src/integration-test/resources/openeIT/__files/test06_getTypeForExistingErrandWithRequestParameter/mappings/api-byggr-read-errand.json:21]: Doesn't seem correct?

File: src/main/java/se/sundsvall/byggrintegrator/api/OpeneXmlResource.java [hunk 2]
```diff
@@ -47,4 +48,13 @@ public Weight getErrandType(
...
 		return byggrIntegratorService.getErrandType(caseNumber);
 	}
+
+	@GetMapping(path = "/cases/type", produces = { APPLICATION_XML_VALUE, APPLICATION_PROBLEM_XML_VALUE })
+	@Operation(summary = "Return xml structure errand type for the errand matching sent in case number")
+	public Weight getErrandTypeWithRequestParameter(
+		@Parameter(name = "municipalityId", description = "Municipality ID", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
+		@Parameter(name = "caseNumber", description = "Case number from ByggR", example = "BYGG 2001-123456") @NotBlank @RequestParam("caseNumber") String caseNumber) {
+
+		return byggrIntegratorService.getErrandType(caseNumber);
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/byggrintegrator/api/OpeneXmlResource.java:58]: Return` ResponseEntity<Weight>`?

File: src/main/java/se/sundsvall/byggrintegrator/api/OpeneHtmlResource.java [hunk 3]
```diff
@@ -47,4 +48,13 @@ public ResponseEntity<String> findNeighborhoodNotificationFiles(
 
 		return ResponseEntity.ok(byggrIntegratorService.listNeighborhoodNotificationFiles(municipalityId, parseDiaryNumber(caseNumberAndEventId), parseEventId(caseNumberAndEventId)));
 	}
+
+	@GetMapping(path = "/neighborhood-notifications/filenames", produces = { TEXT_HTML_VALUE })
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/byggrintegrator/api/OpeneHtmlResource.java:52]: neighbor-consultations ?

**Comment by josc04** [comment]
Acted on PR comments

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**